2/twemoji.js

```diff
@@ -301,9 +304,10 @@ var twemoji = (function (
         // collect them to process emoji later
         allText.push(subnode);
       }
-      // ignore all nodes that are not type 1 or that
+      // ignore all nodes that are not type 1, that are svg, or that
       // should not be parsed as script, style, and others
-      else if (nodeType === 1 && !shouldntBeParsed.test(subnode.nodeName)) {
+      else if (nodeType === 1 && subnode.namespaceURI !== 'http://www.w3.org/2000/svg' &&
+          !shouldntBeParsed.test(subnode.nodeName.toUpperCase())) {
```

Comment:
just for the sake of updating, I'd rather go with `.toLowerCase()` and change the RegExp accordingly (or write the RegExp lower-case and use `i` flag)